src/data_provider/CMakeLists.txt

```diff
@@ -159,8 +163,9 @@ elseif(APPLE)
   target_link_libraries(sysinfo cjson ${SRC_FOLDER}/external/libplist/bin/lib/libplist-2.0.a)
 endif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
 
-target_link_libraries(sysinfo wazuhext)
+target_link_libraries(sysinfo wazuhext urlrequest)
```

Comment:
If now the `sysinfo` library requires `urlrequest`, we should either link this dependency statically or distribute it with the installer. Please make and end-to-end test with an installed agent by sources and packages.

Reply:
`http-request` is a submodule or can obtain it from make deps stage